src/modules/todo-funtionalities.js

```diff
@@ -51,6 +51,7 @@ const addTodo = () => {
 const deleteTodo = (todoId) => {
   todoId = +todoId;
   toDoListsArray = toDoListsArray.filter((book) => todoId !== book.index);
+  // Reindexing
   const arrLength = toDoListsArray.length;
   // eslint-disable-next-line no-plusplus
   for (let i = 0; i < arrLength; i++) {
```

Comment:
- [ ] I think it would be better if you fix your linters errors instead of disabling them for best practices and best practices reasons. I think to fix that you can use the following syntax:
```JavaScript
// Bad
for (let i = 0; i < arrLength; i++)
...

// Good
for (let i = 0; i < arrLength;  i += 1)
....
```